### ISSUE TITLE
[Blazor Preview 6] Possibly leaking memory during intensive javascript interop?

### ISSUE THREAD

**Issue by kiyote** [aspnet/AspNetCore#11333]
While exploring animating a canvas object using Blazor, I noticed that the mono runtime is spitting out updates, even when there should be no allocation happening.

I've tried to strip the problem down minimally here:
https://github.com/kiyote/Station

Every 10 seconds or so I'm seeing these sorts of things being posted:
WASM: GC_MINOR: (Nursery full) time 0.35ms, stw 0.38ms promoted 0K major size: 880K in use: 168K los size: 1024K in use: 16K

The "in use" value for "major size" goes up by 4-5K every time it posts.

tl;dr - Using `requestAnimationFrame` and calling back in to Blazor extremely frequently seems to leak memory?

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us, @kiyote.
@rynowak, @SteveSandersonMS can one of you look into this? Thanks!

**Comment by SteveSandersonMS** [comment]
> WASM: GC_MINOR: (Nursery full) time 0.35ms

It doesn't mean a memory leak. It just mean the GC ran.

If you can observe the browser's memory use growing in an unbounded way (like, can you get it to grow by hundreds of MB?) then you have a memory leak. In that case please let us know, with repro steps. Thanks!
